cmd/ocsp-updater/main.go

```diff
@@ -219,42 +333,51 @@ func main() {
 
 		cac, closeChan := setupClients(c, stats)
 
+		updater, err := newUpdater(
+			auditlogger,
+			stats,
+			clock.Default(),
+			dbMap,
+			cac,
+			// Necessary evil for now
+			c.OCSPUpdater,
+		)
+		cmd.FailOnError(err, "Failed to create updater")
+
+		auditlogger.Info(app.VersionString())
+
 		go func() {
-			// Abort if we disconnect from AMQP
 			for {
-				for err := range closeChan {
-					auditlogger.Warning(fmt.Sprintf(" [!] AMQP Channel closed, aborting early: [%s]", err))
-					panic(err)
-				}
+				err := <-closeChan
```

Comment:
This was the bit i was thinking we were going to make (and hold on to) a dummy AMQPRPCServer